src/pages/Home/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import MovieDetail from '../../components/MovieDetail';
+import MovieList from '../../components/CardList';
+import { getSelectedMovie, getGenresList } from '../../services';
+
+function Home() {
+  const [movieInfo, setMovieInfo] = useState(null);
+  const genresLabelList = getGenresList();
```

Comment:
This function is being called multiple times here, let's move it to the useEffect to avoid that